Fonts.css

```diff
@@ -0,0 +1,344 @@
+
+
+h1 {
+  font-family: Arial;
+  font-size: 40px;
+}
+h2 {
+  font-family: Martial;
+  font-size: 24px;
```

Comment:
You'll want to use REM's (or EM's) for all font-sizes. Remember, 1rem = 16px (or whatever the default the user sets on their computer) so you'll need to do math. 24px is 24/16 = 1.5rem. 32px is 2 rem, etc. Make sure and update all your font-sizes to rems